protocol/messenger_backup_handler.go

```diff
@@ -132,9 +134,20 @@ func (m *Messenger) handleBackedUpProfile(message *protobuf.BackedUpProfile, bac
 				return err
 			}
 			contentSet = true
-			response.AddImages(idImages)
+			response.SetImages(idImages)
+		}
+	}
+
+	var links identity.SocialLinks
+	for _, s := range message.SocialLinkSettings {
+		err = m.handleSyncSocialLinkSetting(*s, func(link *identity.SocialLink) {
+			links = append(links, *link)
+		})
+		if err != nil {
+			return err
 		}
 	}
+	response.SetSocialLinks(links)
```

Comment:
Add only `if len(links) > 0` and set `contentSet` in that case.

Reply:
i don't understand why we need `if len(links) > 0` , could you explain more?

Reply:
If there are no set images and there are no set links, why would we send a signal then?

Reply:
ahh I see that DisplayName is always set, ok, then we can remove `contentSet` completely. It was needed for previous implementation.

Reply:
oh, i didn't see the variable `contentSet` 🤦‍♂️

Reply:
yes, that's why we can remove it (was needed before)

Reply:
fixed ❤️